packages/ui-components/src/components/ComboBox/ComboBox.component.tsx

```diff
@@ -429,59 +450,64 @@ export const ComboBox = ({
   )
 }
 
-ComboBox.propTypes = {
+export type ComboBoxWidth = "full" | "auto"
+
+//eslint-disable-next-line no-unused-vars
+type OnChangeHandler = (value: string) => void
+
+export interface ComboBoxProps {
   /** The aria-label of the ComboBox. Defaults to the label if label was passed. */
-  ariaLabel: PropTypes.string,
+  ariaLabel?: string
   /** The children to Render. Use `ComboBox.Option` elements. */
-  children: PropTypes.node,
+  children?: React.ReactElement<ComboBoxOptionProps> | React.ReactElement<ComboBoxOptionProps>[] | null
```

Comment:
**(Blocking)**

In the past, we didn't allow `null` as an explicit value for prop `children`. Just `ReactNode` (as in other components). I don't think we should set it here. And we could also remove default value `children = null`. React will always handle `children` gracefully and not render anything if `null` or `undefined`.

P.S. Love the stricter prop type validation. Will use as [future improvement](https://github.com/cloudoperators/juno/pull/589/files#r1836547762).

Reply:
Done - removed null